### PR TITLE
feat: Download comments timestamps incrementally & add vote json bundle

### PR DESCRIPTION
**Pull request by thi4go**
This diff adds incremental download for the paginated comments timestamps. It also cleans the remaining `state (vetted/unvetted)` request params that were obsolete.

It also adds vote details route on our api lib, and cooks up a votes json bundle for download from the gui.

### CODE REVIEW

File: src/containers/Comments/Download/hooks.js [hunk 1]
```diff
@@ -15,3 +25,120 @@ export function useDownloadComments(token) {
...
+                comments: {
+                  ...timestamps.comments,
+                  ...resp.comments
+                }
+              });
+              setProgress(
+                (
+                  (Object.keys(timestamps.comments).length * 100) /
+                  commentsLength
+                ).toFixed(2)
```
Comment (victorgcramos) [inline src/containers/Comments/Download/hooks.js]: I think you should move this to a `getProgressPercentage` function, so we improve the readability.

File: src/containers/Comments/Download/hooks.js [hunk 2]
```diff
@@ -15,3 +25,120 @@ export function useDownloadComments(token) {
...
+          setProgress(100);
+          fileDownload(
+            JSON.stringify(
+              {
+                comments: timestamps.comments,
+                serverpublickey: apiInfo.pubkey
+              },
+              null,
+              2
+            ),
```
Comment (victorgcramos) [inline src/containers/Comments/Download/hooks.js]: I think you could move it to another function too, what do you think? Little hard to understand what's going on here just by looking at it :)

File: src/containers/Comments/Download/hooks.js [hunk 3]
```diff
@@ -15,3 +25,120 @@ export function useDownloadComments(token) {
...
+        return;
+      },
+      start: () => {
+        // fetch first page of comments timestamps
+        const [fetch, next] = getCommentIdsForPagination(remaining);
+        onFetchCommentsTimestamps(recordToken, fetch)
+          .then(({ comments }) => {
+            setTimestamps({ comments });
+            setProgress(
+              ((Object.keys(comments).length * 100) / commentsLength).toFixed(2)
```
Comment (victorgcramos) [inline src/containers/Comments/Download/hooks.js]: same here

**Review by victorgcramos — CHANGES_REQUESTED** [review]
Looks good. Nice job using the fetch machine! 🚀

Left some minor comments regarding the code declarativeness.

File: src/containers/Comments/Download/hooks.js (hunk 1 above)
Comment (thi4go) [reply src/containers/Comments/Download/hooks.js]: Good idea mate, done

**Review by victorgcramos — APPROVED** [review]
LGTM

**Pull request merged**